apps/sms/js/thread_ui.js

```diff
@@ -2074,8 +2059,8 @@ var ThreadUI = {
           params.items.push({
             l10nId: 'forward',
             method: () => {
-              this.navigateToComposer({
-                forward: { messageId: messageId }
+              MessageManager.getMessage(messageId).then((message) => {
```

Comment:
IIRC we're going to cache messages in Threads.js? If so - then it will be even more simpler here :)